### ISSUE TITLE
Dontworking with mobile browsers

### ISSUE THREAD

**Issue by kaanc** [davidjbradshaw/iframe-resizer#122]
Hi,
iframe-resizer dont working with mobile browsers(I tested android chrome and android default browser). İframe height is not 100%, it's  minimal size

**Comment by masihur** [comment]
Hi,
I added my forum into using i-frame, but on android default browser height not caluculated, and forum show only in 100px.

**Comment by masihur** [comment]
Hi,
I added i-frame using iframe-resizer, also added some sandbox attribute for i-frame to show i-frame into mobile version, so that our forum not looking like into i-frame. I have used it for the WordPress site. 
I am using native version. But the forum show inside i-frame and could not cover full page.

**Comment by masihur** [comment]
Im using the following lib http://davidjbradshaw.github.io/iframe-resizer/ which enables cross site iFrame's to be resized after source page. 

Although I encountered the following problem on the mobile device Im not able to solve:

When first loading the page the height of the iFrame is right (the whole loaded page is seen in the iframe). But when pressing a link which height is less then previous page then "resizedCallback" function could not called. So the current page height remain same as previous page. While the height of the current page is too small from previous page. This arises only on mobile.

**Comment by davidjbradshaw** [comment]
Please update to v2.8.6 and try again.

**Comment by davidjbradshaw** [comment]
Please also read this about filling bug reports.

https://github.com/davidjbradshaw/iframe-resizer/blob/master/CONTRIBUTING.md

**Comment by masihur** [comment]
I have test it on i-phone as well as android default browser and chrome.
Steps.
I have applied below code for i-frame resizing
iFrameResize({
	log:true,
	interval:1,
	tolerance: 1,
	resizeFrom : 'parent',
	heightCalculationMethod: heightResizeMethod,
	resizedCallback: function(messageData){
			iframeResizerLoaded = true;
	}
});

Also I have applied this code into wordpress site to open our forum inside the i-frame.
Once user clicks on the forum then it calculate the page height and forum page open according to his height.
But once user clicks another page inside i-frame which height is less than previous page then resizedCallback could not call while if we click another page inside i-frame which height is greater than previous page height then it's working.
This issue arises on all mobile (i-phone, android etc)
